pkg/odo/cli/service/create.go

```diff
@@ -227,7 +227,12 @@ func NewCmdServiceCreate(name, fullName string) *cobra.Command {
 		Run: func(cmd *cobra.Command, args []string) {
 			util.LogErrorAndExit(o.Complete(name, cmd, args), "")
 			util.LogErrorAndExit(o.Validate(), "")
-			util.LogErrorAndExit(o.Run(), "")
+			equivalent := o.outputNonInteractiveEquivalent()
+			if len(equivalent) > 0 {
+				log.Info("Equivalent command:")
+				fmt.Println(ui.StyledOutput(equivalent, "cyan"))
+			}
+			//util.LogErrorAndExit(o.Run(), "")
```

Comment:
Was this commented out as part of a test?

Reply:
Ah crap… yes, I was testing the command and wanted to go to the end without actually creating the service. Forgot to clean it up… :( Thanks for catching it up!

Reply:
You're welcome :)